### DISCUSSION TITLE
Achievement Badge

### DISCUSSION THREAD

**Discussion by Om-Vilas-Kulkarni** [community/community#59626] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

**Topic**-Achievement section bug
Hello guys!!, I have 4 pull requests merged but I still get pull shark achievement badge. Please can you help?

In settings my show achievements is also enabled but still I can't see it on my profile, nor I received any mail of after my first 2 pull request merged. I don't know what's the problem. Please help

My username is @Om-Vilas-Kulkarni
![prb2](https://github.com/community/community/assets/114779868/820ea6a1-b1d8-4200-b9bd-a4e168c218e1)
![prb1](https://github.com/community/community/assets/114779868/cec06ab7-cc79-4682-863c-3bc89e62b11d)

**Comment by leeringrose** [comment]
Hey @Om-Vilas-Kulkarni ,
Have you installed any browser extension on your account? if you have a blocker extension installed, it could potentially block the display of badges.

**Reply by Om-Vilas-Kulkarni** [reply]
How to know if blocker extension is installed?? also I didn't receive any mail after 2 pull requests

**Reply by Om-Vilas-Kulkarni** [reply]
then how do i get pull shark achievement?

**Reply by seo-asif** [reply]
Wait 1 day... sometimes It takes time like mine... check my profile

**Comment by ArdeshirV** [comment]
### First changing the visibility of your private contributions
You can choose to publicize your private contributions, which allows visitors to your profile to see private contribution counts, without further details. If you choose to hide your private contributions, visitors will only see your public contributions.
- In the top right corner of GitHub.com, click your profile photo, then click Your profile.
![](https://docs.github.com/assets/cb-18133/mw-1440/images/help/profile/top-right-avatar.webp)
- Under "Contributions & Activity", select or deselect Include private contributions on my profile.

### Second Changing the visibility of achievements
- In the upper-right corner of any page, click your profile photo, then click Settings.
![](https://docs.github.com/assets/cb-65929/mw-1440/images/help/settings/userbar-account-settings.webp)
- Under "Profile settings", select or deselect Show Achievements on my profile.
[![](https://i.postimg.cc/05J0fjPX/Screenshot-from-2023-06-29-17-58-38.png)](https://postimg.cc/HjdMWpJb)
- Click Update preferences.

**Reply by Om-Vilas-Kulkarni** [reply]
@ArdeshirV show achievements is already enabled still can't see

**Reply by ArdeshirV** [reply]
Did you enable private contributions?
[![](https://i.postimg.cc/PJKcfR61/Screenshot-from-2023-06-29-18-34-11.png)](https://postimg.cc/zHbp00Yf)

**Reply by Om-Vilas-Kulkarni** [reply]
yes both are enabled
![Screenshot 2023-06-29 232002](https://github.com/community/community/assets/114779868/104dd154-69b8-4d9b-9c92-04fef45faff5)

**Reply by Om-Vilas-Kulkarni** [reply]
@ArdeshirV both are enabled but still can't see any changes on profile

**Reply by ArdeshirV** [reply]
Now let's do something else.
Fork this simple repository: https://github.com/ArdeshirV/sample
And send two pull-request to above repository. (Do whatever you want)
Then I will accept your pull-requests immediately to check the problem again.

**Reply by Om-Vilas-Kulkarni** [reply]
@ArdeshirV I'm sending you pull request

**Reply by Om-Vilas-Kulkarni** [reply]
@ArdeshirV I have sent the pull request pls check it

**Reply by ArdeshirV** [reply]
### Hope you got your answer now!
[![Now it seems you have a little shark!](https://i.postimg.cc/TP2wPwQ2/Screenshot-from-2023-06-30-22-14-15.png)](https://postimg.cc/75cDsqpj)
### Yeah! Congratulation!

**Reply by Om-Vilas-Kulkarni** [reply]
Damn how come it's showing now??? what was the issue?

**Reply by Om-Vilas-Kulkarni** [reply]
**Thanks buddy!!**

**Reply by Vanessaaui** [reply]
I'm so sorry

**Comment by ArdeshirV** [comment, marked as answer]
> Damn how come it's showing now??? what was the issue?

I told you please send another pull-request to me because I think maybe you need to send your pull-requests into two different repositories to gain a shark, but it seems all your other pull requests were about just one repository. Now you worked with two different repositories; So you achieved the shark badge finally! Anyway congratulation! Hope you got your answer now. Would you like to accept my answer?

**Reply by Om-Vilas-Kulkarni** [reply]
Oh now I understood!, to get pull shark you need to send two pull requests in two different repositories and then you can gain the badge. Thanks a ton buddy!!
